app/models/user.rb

```diff
@@ -50,8 +50,23 @@ def send_activation_email
     UserMailer.account_activation(self).deliver_now
```

Comment:
You are having some syntax error right here. Please look into it.

Reply:
I have changed how this mailer works in order to enable deliver_later, 
```suggestion
    UserMailer.account_activation(self, token).deliver_now
```